### ISSUE TITLE
Manage connected browsers and devices outdated

### ISSUE THREAD

**Issue by geeseven** [nextcloud/documentation#1746]
Greetings,

The [Manage connected browsers and devices](https://docs.nextcloud.com/server/latest/user_manual/en/session_management.html) contain outdated screenshots and is missing information like the 'Allow filesystem access' and 'Wipe device' options.

**Comment by skjnldsv** [comment]
cc @ChristophWurst @rullzer @jospoortvliet

**Comment by ak090498** [comment]
is this still open ?

**Comment by ChristophWurst** [comment]
Yes: https://docs.nextcloud.com/server/latest/user_manual/en/session_management.html

**Comment by ChristophWurst** [comment]
sure!

**Comment by joshtrichards** [comment]
Related: nextcloud/server#40161
